### PR TITLE
JAVA8MIG-742 last successful Pipeline Task field on API

### PR DESCRIPTION
**Pull request by chhex**
Neues Feld in Patch: lastPipelineTask, siehe auch https://github.com/apgsga-it/jenkins-patch-scripts/pull/47

### CODE REVIEW

File: apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/PatchBean.java [hunk 1]
```diff
@@ -313,6 +314,17 @@ public String getTargetToState() {
 	public void setTargetToState(String targetToState) {
 		this.targetToState = targetToState;
 	}
+	
+	
+	@Override
+	public String getLastPipelineTask() {
+		return targetToState;
```
Comment (apgsga-jhe) [inline apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/PatchBean.java]: Shouldn't we return "lastPipelineTask" ?

Comment (chhex) [reply apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/PatchBean.java]: @apgsga-uge  good point

Comment (chhex) [reply apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/PatchBean.java]: @apgsga-jhe @apgsga-uge done

**Comment by chhex** [comment]
@apgsga-jhe @apgsga-uge  I mergeing these change to master.

**Comment by chhex** [comment]
@apgsga-jhe @apgsga-uge with a bumped version.

**Pull request merged**